learning/tour-of-beam/frontend/lib/components/footer.dart

```diff
@@ -26,41 +26,43 @@ import '../constants/links.dart';
 import '../constants/sizes.dart';
 
 class Footer extends StatelessWidget {
-  const Footer({super.key});
+  const Footer();
 
   @override
   Widget build(BuildContext context) {
+    final linkButtonStyle = TextButton.styleFrom(
+      textStyle: const TextStyle(
+        fontWeight: ProjectFontWeights.normal,
+      ),
+    );
+
     return Container(
       color: ThemeColors.of(context).secondaryBackground,
       width: double.infinity,
       child: Padding(
         padding: const EdgeInsets.symmetric(
-          vertical: kSmSpacing,
-          horizontal: kXlSpacing,
+          vertical: ProjectSpacing.small,
```

Comment:
Why do you add "Project"? +7 characters to all of these.

Reply:
If you want a prefix, "App" may be better, it is 4 chars shorter.

Reply:
The prefix is needed to avoid duplicating names like Colors. I replaced Project with Tob, because it's possible for App to be confused with something else.